frontend/src/pages/ClientVenuesPage.jsx

```diff
@@ -1,11 +1,47 @@
-import CheckoutPage from "./CheckoutPage";
+import React from "react";
+import VenueCard from "../components/VenueCard";
+import Saddledome from "../assets/Saddle.jpg";
+import TMobile from "../assets/T-Mobile2.png";
+import Amalie from "../assets/Amalie.jpg";
+import Climate from "../assets/Climate.jpeg";
 
+const data = [
+	{
+		name: "Scotiabank Saddledome",
+		location: "Calgary, AB",
+		img: Saddledome
+	},
+	{
+		name: "T-Mobile Arena",
+		location: "Paradise, NV",
+		img: TMobile
+	},
+	{
+		name: "Amalie Arena",
+		location: "Tampa, FL",
+		img: Amalie
+	},
+	{
+		name: "Climate Pledge Arena",
+		location: "Seattle, WA",
+		img: Climate
+	},
+]
 function ClientVenuesPage() {
 	return ( 
-		<div className="bg-dark text-white min-h-screen items-center"> 
-			<h1 className="pt-36">client venues</h1>
+		<div className="flex flex-col bg-dark min-h-screen w-max text-white items-center justify-center">
+			{/* Page container */}
+			<div className="flex flex-col justify-center gap-1 text-2xl px-12">
+				<h1>AVAILABLE VENUES</h1>
+				{/* Carousel */}
+				<div className="flex flex-row overflow-scroll gap-12 mx-auto">
```

Comment:
<img width="1505" alt="Screenshot 2023-03-28 at 9 52 57 PM" src="https://user-images.githubusercontent.com/65151396/228422506-a3bcd4a7-4374-4ed7-9252-eda2385794de.png">

Minor nit, you can see the scrollbars and whatnot on the carousel (The white bar on the bottom) and there's the scrollbar on the bottom of the page. The only scrollable part should be the **container** not the whole screen, could you fix that?